payment/payment-domain/src/main/java/shop/woowasap/payment/domain/PayStatus.java

```diff
@@ -2,7 +2,9 @@
 
 public enum PayStatus {
 
+    PENDING,
     SUCCESS,
     FAIL,
+    CANCELD,
```

Comment:
요거 오타인거 같아요 (중요하진 않음)

```suggestion
    CANCELED,
```

<img width="678" alt="스크린샷 2023-08-19 오후 2 45 41" src="https://github.com/woowasap/ASAP/assets/62425964/d1d0e810-704a-4e8b-9a49-dfe54cd191e9">